### PR TITLE
docs: implement feedback

### PR DESCRIPTION
**Pull request by guenthermi**
Improve documentation according to feedback
---
Implements the documentation according to the points listed in #523 

- [x] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: docs/walkthrough/basic-concepts.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-Finetuner organizes your training based on two concepts: {class}`~finetuner.experiment.Experiment` and {class}`~finetuner.run.Run`.
+Finetuner organizes your training based on two concepts: 
+{class}`~finetuner.experiment.Experiment` and {class}`~finetuner.run.Run`.
 
 An Experiment defines the machine learning task you're fine-tuning for.
-A Run is a piece of code that performs the Experiment with specific configurations.
-An Experiment contains a list of Runs,
-each with different configurations.
+A Run refers to a single execution of the Experiment with specific configurations.
```
Comment (gmastrapas) [inline docs/walkthrough/basic-concepts.md:8]: a specific configuration.

File: docs/walkthrough/choose-backbone.md [hunk 2]
```diff
@@ -34,12 +35,11 @@ To get a list of supported models:
 └──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴────────────────────────────────────────────────┘
 
 ```
-
-+ ResNets are suitable for image-to-image search tasks with high performance requirement.
-+ EfficientNets are suitable for image-to-image search tasks with fast training and inference. The model is more light-weighted than ResNet.
++ ResNets are suitable for image-to-image search tasks with high performance requirement, where `resnet152` is bigger and requires higher computational resources than `resnet50`.
++ EfficientNets are suitable for image-to-image search tasks with fast training and inference. The model is more light-weighted than ResNet. Here, `efficientnet_b4` is the bigger and more complex model.
```
Comment (gmastrapas) [inline docs/walkthrough/choose-backbone.md:39]: training and inference times.

File: docs/walkthrough/choose-backbone.md [hunk 3]
```diff
@@ -34,12 +35,11 @@ To get a list of supported models:
 └──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴────────────────────────────────────────────────┘
 
 ```
-
-+ ResNets are suitable for image-to-image search tasks with high performance requirement.
-+ EfficientNets are suitable for image-to-image search tasks with fast training and inference. The model is more light-weighted than ResNet.
++ ResNets are suitable for image-to-image search tasks with high performance requirement, where `resnet152` is bigger and requires higher computational resources than `resnet50`.
```
Comment (gmastrapas) [inline docs/walkthrough/choose-backbone.md:38]: performance requirements

File: docs/walkthrough/create-training-data.md [hunk 4]
```diff
@@ -79,7 +83,11 @@ you only need to create a `Document` with `content` and `tags` with the `finetun
 For cross-modality (text-to-image) fine-tuning with CLIP,
 you should create a root `Document` which wraps two {class}`~docarray.array.chunk`s with the `image` and `text` modality.
 The image and text form a pair.
-Evaluation data should be created the same way as above.
+During the training, CLIP learns to place documents which are part of a pair close to
+each other and documents which are not part of a pair far from each other.
+Accordingly, no further labels need to be provided.
```
Comment (gmastrapas) [inline docs/walkthrough/create-training-data.md:88]: As a result, no further

File: docs/walkthrough/create-training-data.md [hunk 5]
```diff
@@ -79,7 +83,11 @@ you only need to create a `Document` with `content` and `tags` with the `finetun
...
+
+Evaluation data should be created the same way as the training data in the examples above.
```
Comment (gmastrapas) [inline docs/walkthrough/create-training-data.md:90]: in the same way

File: docs/walkthrough/save-model.md [hunk 6]
```diff
@@ -2,10 +2,21 @@
 # Save Artifact
 
 Perfect!
-Now you have started the fine-tuning job in the cloud.
-However, fine-tuning takes time. It highly depends on the size of your training data, evaluation data and other hyper-parameters. Because of this, you might have to close the session and reconnect to it several times.
+Now, you have started the fine-tuning job in the cloud.
+When the fine-tuning job is finished, the resulting model is automatically stored under your Jina account in the cloud.
+Next, we can get its hubble artifact id and download the model.
```
Comment (gmastrapas) [inline docs/walkthrough/save-model.md:7]: no need to mention Hubble

File: docs/walkthrough/basic-concepts.md [hunk 7]
```diff
@@ -17,6 +17,14 @@ For example:
   - Run2: Use ResNet50 with learning rate 0.01 and SGD optimizer.
   - Run3: Use ResNet50 with learning rate 0.0001 and Adam optimizer.
 
+All information and data produced during using fine-tuner is linked to those two concepts.
```
Comment (bwanglzu) [inline docs/walkthrough/basic-concepts.md:20]: `fine-tuner` -> `finetuner`

**Review by gmastrapas — APPROVED** [review]
Nice!

File: docs/walkthrough/integrate-with-jina.md [hunk 8]
```diff
@@ -143,3 +143,54 @@ da.post(
...
 Shape of the embedding: (768,)
 ```
+
+## Special case: Artifacts with CLIP models 
+If your fine-tuning job was executed on a CLIP model, your artifact contains two 
+models: `clip-vision` and `clip-text`.
+The vision model allows you to embed images and the text model can encode text passages
+into the same vector space.
+To host those models, you have to provide the name of the model via an additional
+`select_model` property inside the `uses` attribute:
```
Comment (bwanglzu) [inline docs/walkthrough/integrate-with-jina.md:153]: maybe change `property` to `parameter` or `argument`?

**Review by bwanglzu — COMMENTED** [review]
minor comments, in general looks good!

**Pull request merged**